developer/src/kmcmplib/subprojects/gtest.wrap

```diff
@@ -0,0 +1,16 @@
+[wrap-file]
+directory = googletest-1.14.0
```

Comment:
```suggestion
directory = googletest
```

the version number doesn't seem important to be in the directory name (change .gitignore to match)

Reply:
The directory name comes from the meson `.wrap` file, which installs a patched version of googletest ... I don't want to change the `.wrap` file ... I will leave this 'as is'.